feg/cloud/go/go.mod

```diff
@@ -12,6 +12,7 @@
 module magma/feg/cloud/go
 
 replace (
+	magma/feg/cloud/go => ../../../feg/cloud/go
```

Comment:
are you sure. you need it here? Try to remove and test.

Reply:
this PR will not land, but i am working on another PR that modifies something similar and once we do cross references with lte, we need to add this line. Otherwise it shows missing dependencies